exasol_script_languages_container_ci/lib/__init__.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+from typing import Callable
+
+import docker
+from git import Repo
+
+
+def _get_docker_images():
+    docker_client = docker.from_env()
+    exa_images = []
+    try:
+        exa_images = [str(img) for img in docker_client.images.list() if "exa" in str(img)]
+    finally:
+        docker_client.close()
+    return exa_images
+
+
+def print_docker_images(writer: Callable[[str], None]):
+    """
+    Prints all docker images whith "exa" in it's name to stdout.
+    :return: None
+    """
+
+    from inspect import cleandoc
```

Comment:
This import I may would pull up to the top of the module. I know I put it there when I did the suggestion xD sorry :D.